portfolio/src/main/webapp/topten.html

```diff
@@ -15,10 +15,8 @@
 	<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script>
 	<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
 	<script>
-		loadSelector("#nav", "navbar.html");
-	</script>
-  <script>
-		loadSelector("#social", "socials.html");
+	  loadHTML("#nav", "navbar.html");
+		loadHTML("#social", "socials.html");
```

Comment:
nit: the indentation is off here. lines 18-19 should be at the same indent level.